homeassistant/components/maxcube/climate.py

```diff
@@ -117,9 +126,13 @@ def temperature_unit(self):
     def current_temperature(self):
         """Return the current temperature."""
         device = self._cubehandle.cube.device_by_rf(self._rf_address)
-
-        # Map and return current temperature
-        return self.map_temperature_max_hass(device.actual_temperature)
+        if (
+            device.actual_temperature is None
+            or device.actual_temperature < self.min_temp
+            or device.actual_temperature > self.max_temp
```

Comment:
Why do this check, what if the current temperature is higher than the maxcube supports ?

Reply:
Very good point, this is an oversight and I shall remove it.

Reply:
Change has been pushed @balloob